controllers/controllers/networking/cfroute_controller_test.go

```diff
@@ -144,36 +157,19 @@ var _ = Describe("CFRouteReconciler Integration Tests", func() {
 		}).Should(Succeed())
 	})
 
-	When("the route Host contains upper case characters", func() {
```

Comment:
Is it safe to be removing this test?

Reply:
OK - validation stops any uppercase characters being used, so this was redundant.